frontend/control-center/src/pages/Connectors/index.tsx

```diff
@@ -113,15 +114,14 @@ const Connectors = (props: ConnectedProps<typeof connector>) => {
                   )) ||
                 (getSourceForComponent(infoItem.type) &&
                   components &&
-                  components[infoItem.configKey] &&
```

Comment:
this check is needed otherwise it can trigger an error 
<img width="1432" alt="Screenshot 2022-08-15 at 12 40 10" src="https://user-images.githubusercontent.com/38159391/184622887-ffa23b49-0d6b-4655-8628-70b73a726ac4.png">
<img width="1141" alt="Screenshot 2022-08-15 at 12 40 44" src="https://user-images.githubusercontent.com/38159391/184622896-89a703e5-680a-46d8-a6d5-260e07717d4a.png">